docs/cedarling/integrations/cedarling-opa.md

```
@@ -0,0 +1,308 @@
+    cp ../target/release/libcedarling_go.so plugins/cedarling_opa/
+    mkdir -p build
+    export CGO_ENABLED=1
+    export CGO_LDFLAGS="-Lplugins/cedarling_opa"
+    go build -o build/opa-cedarling
+    ```
+
+## Running
+
+1. Set the library path so the plugin can find the Rust binding by running this from the `cedarling_opa` directory:
+
+    ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)